### DISCUSSION TITLE
How to install milestone version?

### DISCUSSION THREAD

**Discussion by ghost** [reduxjs/redux-toolkit#1555] [Q&A]
I tried `@reduxjs/toolkit@next` and `@reduxjs/toolkit@latest`, but got `1.6.0-rc#` or `1.6.1` version, but for me needs `1.6.2` and more. How to install that version with correct source maps?

**Comment by markerikson** [comment]
There is no 1.6.2 release yet.  Per our releases page, 1.6.1 is the latest:

https://github.com/reduxjs/redux-toolkit/releases

**Reply by ghost** [reply]
No, I want install nightly version.

**Reply by markerikson** [reply]
There is no "nightly version". We don't do automated releases. You _can_ install the CodeSandbox CI build from a specific PR if you want.

**Reply by ghost** [reply]
I found two issues. 
- `redux-toolkit.esm.ts` not found for source maps
- `tsc` not working for `npm run build`

**Comment by phryneas** [comment]
You open the PR

https://github.com/reduxjs/redux-toolkit/pull/1459

You scroll down to the comment from CodeSandbox and click "here"

> To see build info of the built libraries, click here or the icon next to each commit SHA.

You scroll to "Local install instruction"

```
yarn add https://pkg.csb.dev/reduxjs/redux-toolkit/commit/f09c978a/@reduxjs/toolkit https://pkg.csb.dev/reduxjs/redux-toolkit/commit/f09c978a/@rtk-query/graphql-request-base-query
npm i https://pkg.csb.dev/reduxjs/redux-toolkit/commit/f09c978a/@reduxjs/toolkit https://pkg.csb.dev/reduxjs/redux-toolkit/commit/f09c978a/@rtk-query/graphql-request-base-query
```
